corehq/apps/export/migrations/0006_delete_dailysavedexportnotification.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.14 on 2018-10-09 19:24
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('export', '0005_datafile_blobmeta'),
+    ]
+
+    operations = [
+        migrations.DeleteModel(
+            name='DailySavedExportNotification',
```

Comment:
> Planning to delete the class in another PR once this one has been merged and deployed on all environments

This migration would delete the table in this PR

Reply:
Removed in d0cd939